2.0/.sti/bin/assemble

```diff
@@ -15,7 +15,6 @@ function rake_assets_precompile() {
 set -e
 
 export RACK_ENV=${RACK_ENV:-"production"}
-export RAILS_ENV=${RAILS_ENV:-"${RACK_ENV}"}
```

Comment:
We did this to make it compatible with v2 cartridge. Is this compatibility off the hook now?